MRETestBed/Assets/TestBed Assets/Shaders/MREDiffuseVertex.shader

```diff
@@ -122,7 +118,7 @@ Shader "MRE/DiffuseVertex" {
 					fixed4(_EmissiveColor.rgb, 0.0) * tex2D(_EmissiveTex, TRANSFORM_TEX(i.uv, _EmissiveTex)) +
 					i.diffuse * tex2D(_MainTex, TRANSFORM_TEX(i.uv, _MainTex));
 				UNITY_APPLY_FOG(i.fogCoord, color);
-				clip(lerp(1, color.a - _AlphaCutoff, _ShouldCutout));
+				clip(lerp(1, color.a, 0));
```

Comment:
[clip](https://developer.download.nvidia.com/cg/clip.html) discards the current pixel if the passed value is less than 0. This lerp expression always evaluates to 1, so is basically a computationally expensive no-op. Though re: my previous comment, we shouldn't be changing this shader at all for this.